gsrs-rest-api/src/main/java/gsrs/api/GsrsEntityRestTemplate.java

```diff
@@ -92,24 +99,29 @@ public ExistsCheckResult exists(String... anyKindOfIdString) throws IOException{
                list.add(s);
            }
        }
-
-        return restTemplate.postForObject(prefix+"/@exists", list, ExistsCheckResult.class);
+       return restTemplate.postForObject(prefix+"/@exists", list, ExistsCheckResult.class);
     }
 
-    public <S extends T> Optional<PagedResult<S>> page(long top, long skip) throws JsonProcessingException {
+    public <S extends T> Optional<PagedResult<S>> page(long top, long skip) throws IOException {
```

Comment:
This may be an API breaking change for others that have implemented this. It still may be a better design, but to try to be backward compatible, can you keep this throwing the same kind of exception for now? Just wrap the current exceptions as JsonProcessingException until there's a chance for everyone to update their given implementations / uses?

Reply:
nah, just make it throw IOException we are still in SNAPSHOT.  I'd rather get it right now while we still "own" all implementations and can fix them without anyone's code breaking